llama_hub/semantic_scholar/README.md

```diff
@@ -0,0 +1,56 @@
+# Semantic Scholar Loader
+
+Welcome to Semantic Scholar Loader. This module serves as a crucial utility for researchers and professionals looking to get scholarly articles and publications from the Semantic Scholar database.
+
+For any research topic you are interested in, this loader reads relevant papers from a search result in Semantic Scholar into `Documents`. 
+
+## Usage
+
+Here is an example of how to use this loader in `llama_index` and get citations for a given query.
+
+### LlamaIndex
+
+```python
+
+from llama_index.llms import OpenAI
+from llama_index.query_engine import CitationQueryEngine
+from llama_index import (
+    VectorStoreIndex,
+    ServiceContext,
+)
+
+s2reader = download_loader('SemanticScholarReader')
```

Comment:
nit: you never import `download_loader` from `llama_index`

also recommended way now is `from llama_hub.semantic_scholar.base import SemanticScholarReader` (e.g. import as package)

Reply:
ahh thanks for pointing that out. I have included it here 20819eae3bf2d7d67aeeaf6f3460e3aa7fe19e4f